### PR TITLE
istioctl: default logs to warn

### PR DESCRIPTION
**Pull request by hanxiaop**
**Please provide a description of this PR:**

### CODE REVIEW

**Review by zirain — APPROVED** [review]

File: istioctl/pkg/root/root.go [hunk 1]
```diff
@@ -49,5 +49,6 @@ func defaultLogOptions() *log.Options {
 	o.SetOutputLevel("klog", log.WarnLevel)
 	o.SetOutputLevel("kube", log.ErrorLevel)
 	o.SetOutputLevel("controllers", log.WarnLevel)
+	o.SetOutputLevel("util", log.WarnLevel)
```
Comment (howardjohn) [inline istioctl/pkg/root/root.go:52]: should we just change everything to warn, and then opt-in others that we want at different levels

Comment (hzxuzhonghu) [reply istioctl/pkg/root/root.go:52]: That would be easy to maintain

Comment (hanxiaop) [reply istioctl/pkg/root/root.go]: Makes sense

**Review by howardjohn — APPROVED** [review]
LGTM

**Pull request merged**